utils/update-checkout

```diff
@@ -52,6 +53,21 @@ def update_working_copy(repo_path):
         else:
             check_call([ "svn", "update" ])
 
+def obtain_additional_swift_sources():
```

Comment:
This is outside the scope of what this script is meant for. This script updates current sources.. At this point you are getting into package management.. There are separate utils for this within swift.